src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -644,9 +646,14 @@ private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
                             int startParam = i;
                             i++;
                             boolean escaped = false;
-                            while (((i + 1) < c.length) && !(!escaped && (c[i] == '"') && (c[i + 1] == ')'))) {
+                            while (((i + 1) < c.length)
```

Comment:
Even if there is no suitable method which can be reused, it may be worthwhile to extract some of the code here to StringUtils.